.env

```
@@ -127,12 +127,38 @@ _APP_GRAPHQL_MAX_COMPLEXITY=250
 _APP_GRAPHQL_MAX_DEPTH=4
 _APP_DOCKER_HUB_USERNAME=
 _APP_DOCKER_HUB_PASSWORD=
-_APP_VCS_GITHUB_APP_NAME=
-_APP_VCS_GITHUB_PRIVATE_KEY=disabled
-_APP_VCS_GITHUB_APP_ID=
-_APP_VCS_GITHUB_CLIENT_ID=
-_APP_VCS_GITHUB_CLIENT_SECRET=
-_APP_VCS_GITHUB_WEBHOOK_SECRET=
+_APP_VCS_GITHUB_APP_NAME=appwrite-locally-final
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The _APP_VCS_GITHUB_APP_NAME key should go before the _APP_WORKER_PER_CORE key

(UnorderedKey)